pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -318,26 +334,104 @@ func (c *podIPMetricsCollector) setupDirectoryWatcher(dir string) error {
 	if err != nil {
 		glog.Errorf("Failed to add watcher for directory %s: %v", dir, err)
 	}
-	podIPMetricsWatcherSetup = true
+	podIPMetricsWatcherIsIntialized = true
+	return nil
+}
+
+func (c *podIPMetricsCollector) countIPsFromRange(subnet types.IPNet) (uint64, error) {
```

Comment:
The comment from https://github.com/kubernetes/kubernetes/blob/3cf9f66e90d560ac080687610933c712bcf37b39/pkg/registry/core/service/ipallocator/ipallocator.go#L496-L522 is informative as it makes the code more readable - would be great to keep it as well.

Reply:
Done.